ert3/config/_ensemble_config.py

```diff
@@ -0,0 +1,53 @@
+import importlib
+from typing import List, Callable, Optional
+from pydantic.dataclasses import dataclass
+from pydantic import validator
+
+
+class _EnsembleConfig:
+    validate_all = True
+    validate_assignment = True
+    extra = "forbid"
+    arbitrary_types_allowed = True
+
+
+@dataclass(config=_EnsembleConfig)
+class Stages:
+    name: str
+    python_function: Callable
+
+    @validator("python_function", pre=True)
+    def function_is_valid(cls, method: str) -> Callable:
+        return import_from(method)
+
+
+@dataclass(config=_EnsembleConfig)
+class ForwardModel:
+    stages: List[Stages]
+    driver: str = "local"
```

Comment:
Does this default to `local` or imply that the only valid value is `local`? Could we use a literal type instead?

Reply:
Defaults to `local`, there is no validation outside that is has to be a `str` at the moment.